docs/container.md

```diff
@@ -63,12 +64,12 @@ class Config extends ContainerConfig
     {
         $adr = $di->get('radar/adr:adr');
 
+        $adr->middle('Relay\Middleware\ResponseSender');
```

Comment:
After adding this I got another question regarding the order.

 ```php
 $boot = new Boot();
 $adr = $boot->adr([
     'Foo\Bar\ContainerConfig',
     'My\Config',
 ]);
 ```

Does this impact the order of execution ? See the `Foo\Bar\ContainerConfig` is loaded first. So will 

```php
$adr->middle('My\Config\SomeHandler');
```

will be first one in the modify call right ?